### DISCUSSION TITLE
how to change the size of the logo?

### DISCUSSION THREAD

**Discussion by camilomatiz116** [woosignal/flutter-woocommerce-app#54] [General]
Hello

I've been looking for a way to change the size of the logo when the app starts.

can you help me?

**Comment by agordn52** [comment]
Hi @camilomatiz116,

You can update the logo by navigating to this file `lib/resources/widgets/woosignal_ui.dart`.
Then, update the `StoreLogo` widget's height/width.

Hope that helps.

Thanks,
Anthony
